redbot/core/core_commands.py

```diff
@@ -3969,7 +3969,7 @@ async def localblocklist_remove(
         self, ctx: commands.Context, *users_or_roles: Union[discord.Member, discord.Role, int]
     ):
         """
-        Removes user or role from blocklist.
+        Removes user or role from local blocklist.
```

Comment:
IIRC this was done on purpose as @Kowlin  wanted all mentions to the allowlist and denylist at a local level to be just "blocklist" and "allowlist" and references to the global one to be "global allowlist" and "global blocklist"

Reply:
Check above in line 388, there it is also referenced as "local" so consistency is definitely an issue.

Reply:
> IIRC this was done on purpose as @Kowlin wanted all mentions to the allowlist and denylist at a local level to be just "blocklist" and "allowlist" and references to the global one to be "global allowlist" and "global blocklist"

Alright i will get rid of that, do i also remove the one from line 388? and i'll push that too.

Reply:
Feel free and just leave it @Fabian-Evolved, that's something for me to do when I make a gigantic pass over all the wording later in the year.